docs/cgroup-v2.md

```diff
@@ -1,6 +1,6 @@
 # cgroup v2
 
-runc supports cgroup v2 (unified mode) experimentally since v1.0.0-rc91.
+runc supports cgroup v2 (unified mode) experimentally since v1.0.0-rc91, officially since v1.0.0-rc93.
```

Comment:
I don't think it makes much sense to mention experimental support and older release, IOW

```suggestion
runc fully supports cgroup v2 (unified mode) since v1.0.0-rc93.
```

or something like that

Reply:
yet better
```suggestion
runc fully supports cgroup v2 (unified mode) since v1.0.0-rc93. For more details,
see [docs/cgroup-v2.md](./docs/cgroup-v2.md).

Reply:
> see [docs/cgroup-v2.md](./docs/cgroup-v2.md)

This file itself is docs/cgroup-v2.md